crates/sargon/src/wrapped_radix_engine_toolkit/low_level/execution_summary/execution_summary.rs

```diff
@@ -110,6 +110,37 @@ impl ExecutionSummary {
     }
 }
 
+impl ExecutionSummary {
+    pub fn classify_delete_accounts_if_present(&mut self) {
+        if self.detailed_classification.is_empty() {
```

Comment:
prefer return early to avoid "pyramid of doom" (indentation)

Reply:
why are we only checking when there is no other detailed classification btw?

Reply:
we don't want to override the RET classification

Reply:
but we are pushing, not overriding. Won't the manifest be classified with RET before doing this manual check ourselves?

Reply:
Hosts use `.first` on classifcation. I.e. Hosts does not care about multiple classifcations, always reads the first.

Reply:
https://github.com/radixdlt/babylon-wallet-ios/blob/ef47ccba90a8ef35eacd48a505212c49c95580f3/RadixWallet/Core/SargonExtensions/ExecutionSummary%2BExtensions.swift#L5

Reply:
@matiasbzurovski I meant that if the manifest was classified by RET, avoid doing custom classification.